projects/rich-previews/src/google-product-desktop/google-product-desktop.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Google - Product</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="google-product-desktop.css">
+  </head>
+
+  <body>
+    <div id="app">
+      <h1 class="iframe-heading" :class="{ 'iframe-heading--dark-mode': isDarkMode }">{{ type }}</h1>
+      <div v-if="!hasRequiredData" class="warning-message" :class="{
+        'warning-message--web-app': platform === 'web-app',
+        'warning-message--dark-mode': isDarkMode
+      }">
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 100">
+          <path d="M98 85.6L53.9 9.7c-.5-1.4-2.4-2.4-3.8-2.4s-3.3.9-4.3 2.4L1.7 85.6c-1.4 3.3.9 7.1 4.3 7.1h87.7c3.8 0 6.2-3.8 4.3-7.1zm-87.7 0l39.8-68.8L90 85.6H10.3z"/>
+          <path d="M50 16.8L10.2 85.6h79.7L50 16.8zm4.7 66.4h-9.5v-9.5h9.5v9.5zm-9.4-19V35.8h9.5v28.5h-9.5z"/>
+        </svg>
+        <p>This page does not contain the necessary metadata to create a preview.</p>
+      </div>
+
+      <template v-else>
+        <div class="scroll-container">
+          <div class="scroll-container__inner">
+            <div class="product">
+              <div class="product__wrapper">
+                <img :src="image" class="product__image" />
+                <div class="product__content">
+                  <a rel="noopener" target="_blank" class="product__name">{{ name }}</a>
+                  <div>
+                    <span class="product__price">{{ offerPrice }}</span
+                    ><a rel="noopener" target="_blank" class="product__merchant-name">{{ offerSellerName }}</a
```

Comment:
The newline in `</span>` end tag looks a bit strange, or is there a reason behind it?

Reply:
Dit heb ik gedaan om de extra spaties van `inline` te vermijden.

Reply:
Is het niet met CSS op te lossen?